app/models/micropost.rb

```diff
@@ -2,7 +2,7 @@ class Micropost < ApplicationRecord
   belongs_to :user
   has_one_attached :image
   scope :latest, ->{order created_at: :desc}
-  scope :with_user, -> id {where user_id: id}
+  scope :with_user, ->(a, id){where "user_id IN (?) OR user_id = ?", a, id}
```

Comment:
sao không đưa hết vào trong IN em nhỉ mà phải tách ra như vậy, với đổi tên tham số a truyền vào cho có ý nghĩa đi em